File: ws/collaguazo/u1/ws05chickenfarmsimulator/model/egg.java
package ec.edu.ec.chickenfarmsimulator.model;

/**
 *
 * @author Pablo Collaguazo
 */
public class Egg {
    private char size;

    @Override
    public String toString() {
        return "Egg{" + "size= " + size + '}';
    }
    public Egg(char size){
        this.size = size;
    }
          
    public char getSize() {
        return size;
    }

    public void setSize(char size) {
        this.size = size;
    }

    
}

File: ws/cevallos/u1/ws06ChickenFarmJson/python/model/egg.py
class Egg:

    def __init__(self, size):
        self.size = size

    def __str__(self):
        return f"Egg {{size={self.size}}}"
    
    def _get_size(self):
        return self.size
    
    def _set_size(self, size):
        self.size = size
